filebeat/docs/modules-getting-started.asciidoc

```diff
@@ -81,20 +84,28 @@ example:
 ----------------------------------------------------------------------
 ./filebeat -e -modules=system,nginx,mysql
```

Comment:
Yah, I haven't changed much in the getting started because it sounds like our plans for the getting started experience are still in flux. But since you mentioned it, beta 2 users might find the discrepancy a little confusing, so maybe I should go ahead and change them now.